contracts/interchain-token-service/src/contract/query.rs

```diff
@@ -34,6 +34,30 @@ pub fn all_its_contracts(deps: Deps) -> Result<Binary, Error> {
     to_json_binary(&contract_addresses).change_context(Error::JsonSerialization)
 }
 
+pub fn its_chains(deps: Deps, filter: Option<msg::ChainFilter>) -> Result<Binary, Error> {
+    let state_configs = state::load_chain_configs(deps.storage).change_context(Error::State)?;
```

Comment:
the load_chain_configs should make use of pagination as well. Loading all configs from storage could lead to gas overflow later on